### ISSUE TITLE
Unable to request builds with Jenkins 2.60.3

### ISSUE THREAD

**Issue by saselberg** [tomkuijsten/vsjenkinsmanager#48]
So, I'm quite new to Jenkins so it's totally possible that I've got something misconfigured.

I installed the Jenkins Manager, pointed it to my jenkins server and I got a list of jobs.  I right clicked on one and told it to build, and I received the following error: 

The remote server returned an error: (403) Forbidden.
Build job failed with response "No valid crumb was included in the request".

There appears to be a discussion about the .crumb header being renamed to Jenkins-Crumb and not doing so can generate this error.   But that could be totally off the mark.

Any thoughts?

**Comment by pvandervelde** [comment]
No, I think your configuration is probably correct. The current version of the Jenkins Manager does not handle crumbs correctly I think. I would have to have a look at the code. I only took over the software a little while ago but I've been too busy with work to fix some of the issues we're seeing.
